microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -56,236 +52,459 @@ The tutorial steps below walk you through this environment setup.
 
-   ![The 'mapped file' icon, a page with two-way arrows](./index-images/workspaces-workspaces-demo-elements-styles-css.png)
+   ![The 'mapped file' icon, a page with two-way arrows](./index-images/elements-styles-css.png)
 
 1. Change the value of the `color` property of the `<h1>` element to orange.  To do this, select the `<h1>` element in the **DOM Tree**.  In the CSS rule for `h1`, click `fuchsia`, start typing **orange**, and then select **orange** from the color list:
 
-   ![Changing the color property in styles.css](./index-images/workspaces-workspaces-demo-elements-styles-css-pick-color.png)
+   ![Changing the color property in styles.css](./index-images/elements-styles-css-pick-color.png)
 
-1. Open the copy of `styles.css` that's in your `app` directory in a text editor, such as Visual Studio Code.  The `color` property is now set to the new color, which is orange in this example.  The change was not only made in the copy of the file returned from the web server; the change was also made in your mapped file in your `app` workspace directory.
+1. Select the **Sources** tool, right-click `styles.css`, and then select **Open in containing folder**.
+
+   File Explorer or Finder opens.
+
+1. Open the copy of `styles.css` that's in your `/Demos/workspace/` directory in a text editor, such as Visual Studio Code.  The `color` property is now set to the new color, which is orange in this example.  The change was not only made in the copy of the file returned from the web server; the change was also made in your mapped file in your `/Demos/workspace/` workspace directory.
 
 1. Refresh the page.
 
 The color of the `<h1>` element is still set to the new color.  The change remains across a refresh, because when you made the change, DevTools saved the change to disk.  When you refreshed the page, your local server served the modified copy of the file from disk.
 
 **Tip:** You can also change the color by clicking the fucshia-colored swatch to open the color picker to pick a new color. The HEX value for the color you pick is the color name.
 
+Continue with the next section.
+
 
 <!-- ====================================================================== -->
-## Edit HTML and save changes to the source file
+## Step 3: Save an HTML change to disk
+<!-- https://developer.chrome.com/docs/devtools/workspaces/#html -->
+
+Next, edit HTML and save changes to the source file.
 
 In the **Elements** tool, it's possible to change HTML tagging in a copy of the file that's returned by the server.  However, to save your edits to a local source file, you need to use the **Sources** tool instead of the **Elements** tool.
```

Comment:
```suggestion
In the **Elements** tool, it's possible to change the DOM representation of the HTML document, which the browser engine constructed when it loaded the page.  However, changes to the DOM can't be mapped back to HTML source files.  To make changes to an HTML source file, you need to use the **Sources** tool instead of the **Elements** tool.
```